.circleci/docker/Dockerfile

```diff
@@ -1,13 +1,13 @@
 # This requires cuda & cudnn packages pre-installed in the base image.
```

Comment:
Hey @yeounoh Can you please take a pass at the updated Dockerfile? I built a new one it can pass the tests now.